ui/appui-react/src/appui-react/childwindow/CopyStyles.ts

```diff
@@ -2,39 +2,34 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-/**
- * Copies the source CSS into the destination
- * @param targetDoc - target document
- * @param sourceDoc - source document
+
+/** Copies the CSS style sheets from source document into the target document.
  * @internal
  */
 export function copyStyles(
   targetDoc: Document,
   sourceDoc: Document = document
 ) {
-  const stylesheets = sourceDoc.adoptedStyleSheets
-    ? Array.from([...sourceDoc.styleSheets, ...sourceDoc.adoptedStyleSheets])
-    : Array.from(sourceDoc.styleSheets);
+  const styleSheets = Array.from(sourceDoc.styleSheets);
+  styleSheets.forEach(({ ownerNode }) => {
+    // Copy `link` and `style` elements.
+    if (!ownerNode) return;
+    const clonedNode = targetDoc.importNode(ownerNode, true);
+    targetDoc.head.appendChild(clonedNode);
```

Comment:
Have you tested whether `<link>` elements reuse the same cached resource or make a new network request?

Reply:
Yes, it seems like the browser memory cache is reused.